src/Repository/EventRepository.php

```diff
@@ -515,6 +515,26 @@ public function queryCountByMonth(Adherent $referent, int $months = 5): QueryBui
         ;
     }
 
+    public function countParticipantsInReferentManagedArea(Adherent $referent): int
+    {
+        $this->checkReferent($referent);
+
+        $results = $this->createQueryBuilder('event')
+            ->select('event.participantsCount')
+            ->innerJoin('event.referentTags', 'tag')
+            ->where('tag IN (:tags)')
+            ->andWhere('event.committee IS NOT NULL')
+            ->andWhere("event.status = '".Event::STATUS_SCHEDULED."'")
+            ->andWhere('event.participantsCount > 0')
+            ->groupBy('event.id')
+            ->setParameter('tags', $referent->getManagedArea()->getTags())
+            ->getQuery()
+            ->getArrayResult()
+        ;
+
+        return array_sum(array_column($results, 'participantsCount'));
```

Comment:
I think it's better to do this stuff with sgbd to avoid the data transfert between db and php.

Reply:
Agree I'll see if I can do something better but not sure.